tests/test_octodns_provider_ultra.py

```diff
@@ -1,3 +1,12 @@
+from __future__ import unicode_literals
+
+try:
+    # Python 3
+    from urllib.parse import parse_qs
+except ImportError:
+    # Python 2
+    from urlparse import parse_qs
```

Comment:
`six` is used in other places to solve this and it'd be preferable to use it so that it's more easily found and cleaned up when 2.7 support is completely removed. 

```suggestion
from six.moves.urllib.parse import urlparse
```